crates/explorer/src/index.css

```diff
@@ -15,13 +15,19 @@ body {
 }
 
 #wat {
-  width: 50%;
+  width: 33%;
+  height: 100%;
+  overflow: scroll;
+}
+
+#clif {
+  width: 33%;
   height: 100%;
   overflow: scroll;
 }
 
 #asm {
-  width: 50%;
+  width: 33%;
```

Comment:
I don't think we want to apply this CSS all the time, only if we have CLIF to view. If we are doing `wasmtime explore` but using Winch instead of Cranelift, there won't be any CLIF to display, so it doesn't make sense to reserve a third of the screen for it.

I think we can resolve this by using flexbox instead of width percents. That way, if there are three things inside the container, they will each be a third of the width, and if there are two, they will each be half the width.